_docs/stub-metadata.md

```diff
@@ -71,19 +83,28 @@ POST /__admin/mappings/find-by-metadata
 }
 ```
 
+{% endcodetab %}
+
+{% endcodetabs %}
+
 ## Remove stubs by metadata
 
 Similarly, stubs with matching metadata can be removed:
 
-Java:
+{% codetabs %}
+
+{% codetab Java %}
 
 ```java
 removeStubsByMetadata(matchingJsonPath("$.singleItem", containing("123")));
 ```
 
-API:
+{% endcodetab %}
+
+{% codetab JSON %}
 
 ```json
+
 POST /__admin/mappings/remove-by-metadata
 
```

Comment:
Needs to be moved above the code block, but can be kept under the tab. This is a historical issue an not related to the PR

```suggestion

POST /__admin/mappings/remove-by-metadata

```json
```